src/ts_config.h

```diff
@@ -13,7 +13,7 @@
  *
  * This option enables the legacy C++ interface of the
  * ThingSet protocol library. Enable if your C++ code uses
- * DataNode or ArrayInfo instead of ThingSetDataNode or ThingSetArrayInfo.
+ * DataObject or ArrayInfo instead of ThingSetDataObject or ThingSetArrayInfo.
```

Comment:
This should stay DataNode. It is for legacy code.

Reply:
Good spot. This should have been excluded from find & replace action. Fixed in the original commit.